extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -254,19 +282,28 @@ class AmpAccordion extends AMP.BaseElement {
     } else { // Toggle without animation
       this.mutateElement(() => {
         if (toExpand) {
+          if (!section.hasAttribute('expanded')) {
```

Comment:
Why do we need this if check? Isn't this already checked for in the initialization of `toExpand`?

Reply:
toExpand takes into consideration of `opt_forceExpand` (my assumption is that we don't want trigger 'expand' event if the section is already expanded). I can definitely use `isSectionClosedAfterClick` but the name is quite unintuitive.